README.md

```diff
@@ -1,9 +1,48 @@
 # CyberSecurity Scoring System (CSSS)
 
+|Branch|Status (Windows Build)|Status (Linux Build)|Last Commit|
+|---|---|---|---|
+|Master|[![AppVeyor Build status](https://img.shields.io/appveyor/ci/stuajnht/CSSS?label=AppVeyor%20Build)](https://ci.appveyor.com/project/stuajnht/csss)|[![Build Status](https://img.shields.io/travis/stuajnht/CSSS.svg?label=Travis%20Build)](https://travis-ci.org/stuajnht/CSSS) |[![Last Master Commit](https://img.shields.io/github/last-commit/stuajnht/CSSS.svg)]()
+|Develop|[![AppVeyor Build status](https://img.shields.io/appveyor/ci/stuajnht/CSSS/develop?label=AppVeyor%20Build)](https://ci.appveyor.com/project/stuajnht/csss/develop)|[![Develop Build Status](https://img.shields.io/travis/stuajnht/CSSS/develop?label=Travis%20Build)]()|[![Last Develop Commit](https://img.shields.io/github/last-commit/stuajnht/CSSS/develop.svg)]()
+
+[![.Net Versions](https://img.shields.io/badge/.Net%20Versions-4.0|4.5|4.6|4.7-blue.svg)]()
+[![Status](https://img.shields.io/badge/status-Pre%20Release-red.svg)]()
+[![Releases](https://img.shields.io/github/v/release/stuajnht/CSSS?include_prereleases)]()
+[![Maintenance](https://img.shields.io/maintenance/yes/2019.svg)]()
+[![Known Vulnerabilities](https://snyk.io/test/github/stuajnht/CSSS/badge.svg?targetFile=CSSS.sln)](https://snyk.io/test/github/stuajnht/CSSS?targetFile=CSSS.sln)
+[![Contributors](https://img.shields.io/github/contributors/stuajnht/CSSS.svg)]()
+[![Forks](https://img.shields.io/github/forks/stuajnht/CSSS.svg)]()
+[![Issues](https://img.shields.io/github/issues/stuajnht/CSSS.svg)](https://github.com/Stormy102/Lancer/issues)
+
 CyberSecurity Scoring System (CSSS) is a program to assist with training for the CyberPatriot / CyberCenturion competitions.
 
 While it is currently possible to create [your own practice images](https://www.uscyberpatriot.org/competition/training-materials/practice-images), there is only a scoring client for Windows computers. To assist with training competitors with Windows, Ubuntu and Debian Operating Systems, this program has been created to provide a scoring system for them.
 

```

Comment:
Could the build status table go after the introductory paragraphs, rather than at the top of the readme, please? The status badges can stay where they are.